### ISSUE TITLE
is the bot on slumbot.com the winner of the 2018 Annual Computer Poker Competition

### ISSUE THREAD

**Issue by zhoujz10** [ericgjackson/slumbot2017#3]
HI, is the bot on slumbot.com the same as the bot which won the 2018 Annual Computer Poker Competition? THX!  @ericgjackson

**Comment by zhoujz10** [comment]
Could anyone help me on this question? @ericgjackson  Thanks.

**Comment by ericgjackson** [comment]
Yes, it's the same, or almost the same.  The website might have one or two changes incorporated after the competition.
